tool/tctl/common/auth_command.go

```diff
@@ -394,6 +410,23 @@ func (a *AuthCommand) ListAuthServers(ctx context.Context, clusterAPI auth.Clien
 	return nil
 }
 
+// GenerateCRLForCA generates a certificate revocation list for a certificate
+// authoritity.
+func (a *AuthCommand) GenerateCRLForCA(ctx context.Context, clusterAPI auth.ClientI) error {
+	certType := types.CertAuthType(a.caType)
+	if err := certType.Check(); err != nil {
+		return trace.Wrap(err)
+	}
+
+	crl, err := clusterAPI.GenerateCertAuthorityCRL(ctx, certType)
```

Comment:
Don't we need to align the RBAC role check ? 
https://github.com/gravitational/teleport/blob/6973cd652720011c2fb6466cfe0889533d9dfae8/lib/auth/auth_with_roles.go#L5549